### PR TITLE
add support for LMB additional information field

### LINKED ISSUE #18: Add <AdditionalInfo> to integration for campaignID
Client and lead vendor has asked for the following:

Please allow the usage of the tag <AdditionalInformation>  to pass campaignID.

https://drive.google.com/drive/u/1/folders/0B36C7G4bQKsANVBIV0J3dVpHVWs

This is the primary place that the vendor has requested to dynamically adjust the campaignID based on lead scoring data. 

https://activeprospect.zendesk.com/agent/tickets/7029

### PR DESCRIPTION
**Pull request by zalary**
resolves #18

### CODE REVIEW

**Comment by zalary** [comment]
@cgrayson or @alexkwolfe mind taking a look at this before it gets lost in Quarterly meeting shuffle?

**Comment by cgrayson** [comment]
Looking

**Comment by cgrayson** [comment]
This is for #18, right?

**Comment by zalary** [comment]
oh yes, sorry, should have linked that

File: src/inbound.coffee [hunk 1]
```diff
@@ -112,6 +112,10 @@ request = (req) ->
       state: lead.PropertyState
       county: capitalizeWords(lead.PropertyCounty)
 
+  lowermybills:
+    additional_information: lead.AdditionalInformation
```
Comment (cgrayson) [inline src/inbound.coffee]: So `lowermybills.additional_information` will also be created as a new system field, right?

Comment (zalary) [inline src/inbound.coffee]: It seems that it already has been.  This integration has validation for system fields and it passed (which I thought was strange, so  I downloaded lc-fields and it's there

**Comment by cgrayson** [comment]
Assuming that field/namespace is ok, LGTM

File: src/inbound.coffee (hunk 1 above)
Comment (cgrayson) [inline src/inbound.coffee]: :astonished: Well okay then :sunglasses:

**Pull request merged**